pkg/install/resources.go

```diff
@@ -219,7 +220,8 @@ func AllResources(o *VeleroOptions) (*unstructured.UnstructuredList, error) {
 	// Set the GVK so that the serialization framework outputs the list properly
 	resources.SetGroupVersionKind(schema.GroupVersionKind{Group: "", Version: "v1", Kind: "List"})
 
-	for _, crd := range CRDs() {
+	for _, crd := range crds.CRDs {
+		crd.SetLabels(labels())
```

Comment:
Why are we setting the labels here?

Reply:
`velero install` previously created the CRDs with the `component: velero` label. The generated CRDs don't have that, so we need to add them here. I like the idea of everything `velero install` having that label.

Reply:
Gotcha, and agreed.